### PR TITLE
Update footer.mdx

### PR DESCRIPTION
**Pull request by JennySanchez**
cap F for footer for intro and design spec sections

### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by JennySanchez** [comment]
@jeffchew Ready for your review so we can merge the update to the live site asap. Thank you.

File: src/pages/components/footer.mdx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 <PageDescription>
 
-The footer is a section that sits at the bottom of any IBM.com page, and acts as the catch&mdash;all section that helps users navigate and provides corporate level general and legal information. The footer is required on all pages on IBM.com.
+The Footer is a section that sits at the bottom of any IBM.com page, and acts as the catch&mdash;all section that helps users navigate and provides corporate level general and legal information. The Footer is required on all pages on IBM.com.
 
-There are two different footers. All pages are required to use the default footer, with the exception of a few pages that need to minimize distraction, such as checkout process. Please [contact us](https://cognitive-app.slack.com/archives/C2PLX8GQ6) if you think your page needs approval to use the minimal footer. 
+There are two different Footers. All pages are required to use the Default Footer, with the exception of a few pages that need to minimize distraction, such as checkout process. Please [contact us](https://cognitive-app.slack.com/archives/C2PLX8GQ6) if you think your page needs approval to use the Minimal Footer.
```
Comment (jeffchew) [inline src/pages/components/footer.mdx:12]:
```suggestion
There are two different Footers. All pages are required to use the default Footer, with the exception of a few pages that need to minimize distraction, such as checkout process. Please [contact us](https://cognitive-app.slack.com/archives/C2PLX8GQ6) if you think your page needs approval to use the minimal Footer. 
```

I think the `Footer` itself is capital, but the types are not capital? @wonilsuhibm ?

Comment (JennySanchez) [reply src/pages/components/footer.mdx:12]: I think you are right, Jeff. Wonil, can you confirm?

Comment (wonilsuhibm) [reply src/pages/components/footer.mdx:12]: Re types, yes, I agree.

**Comment by wonilsuhibm** [comment]
In the intro copy, is it "The Footer is a section that sits..." or "Footer is a section that sits...". I'm wondering if it's the latter now that we are capitalizing the name of the component.

**Comment by JennySanchez** [comment]
https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/788#issuecomment-744709206

Agreed and will update it.

**Review by jeffchew — APPROVED** [review]

**Pull request merged**